fineract-provider/build.gradle

```diff
@@ -85,14 +86,15 @@ apply plugin: "org.hidetake.swagger.generator"
 dependencyManagement {
     imports {
         mavenBom 'org.springframework:spring-framework-bom:5.2.6.RELEASE'
+        mavenBom 'org.springframework.security:spring-security-bom:5.3.2.RELEASE'
```

Comment:
Question for other reviewers (I honestly don't know): Does the `spring-framework-bom` not already include `spring-security-bom` ? Also, if we are fixing the version of `spring-security-oauth2` below anyway, then what do we need to use this BOM for at all?